package.json

```diff
@@ -219,33 +219,33 @@
     "projen": "npx projen"
   },
   "devDependencies": {
-    "@aws-cdk/aws-iam": "^1.134.0",
-    "@aws-cdk/core": "^1.134.0",
-    "@types/jest": "^27.0.3",
+    "@aws-cdk/core": "^1.148.0",
+    "@types/jest": "^27.4.1",
     "@types/node": "^14.17.0",
     "@typescript-eslint/eslint-plugin": "^5",
     "@typescript-eslint/parser": "^5",
-    "aws-cdk": "^1.134.0",
-    "aws-cdk-github-oidc": "^0.0.21",
-    "aws-sdk": "^2.1043.0",
+    "aws-cdk": "^1.148.0",
```

Comment:
Do you know why this still has a devDependency on v1?

Reply:
Try to manually remove it and run projen again to see if it is added back.

Reply:
i think i found the issue. and also another one with build.
will push changes in a few minutes